miio/device.py

```diff
@@ -243,7 +243,7 @@ def send(self, command: str, parameters: Any=None, retry_count=3) -> Any:
             data, addr = s.recvfrom(1024)
             m = Message.parse(data, token=self.token)
             self._device_ts = m.header.value.ts
-            self._device_id = m.header.value.device_id
+            self._device_id = binascii.hexlify(m.header.value.device_id)
```

Comment:
@rytilahti Does the device_id change over time? I hope not. I assume this line could be removed.